rector.php

```diff
@@ -1,20 +1,18 @@
 <?php
 
 declare(strict_types=1);
+use Rector\DogFood\Rector\Closure\UpgradeRectorConfigRector;
 
 use Rector\Config\RectorConfig;
 
-use Rector\Core\Configuration\Option;
 use Rector\Php55\Rector\String_\StringClassNameToClassConstantRector;
 use Rector\Set\ValueObject\LevelSetList;
 use Rector\Set\ValueObject\SetList;
 
 return static function (RectorConfig $rectorConfig): void {
-    $parameters = $rectorConfig->parameters();
-
-    $parameters->set(Option::AUTO_IMPORT_NAMES, true);
-    $parameters->set(Option::PATHS, [__DIR__ . '/src', __DIR__ . '/tests']);
-    $parameters->set(Option::SKIP, [
+    $rectorConfig->importNames();
+    $rectorConfig->paths([__DIR__ . '/src', __DIR__ . '/tests']);
+    $rectorConfig->skip([
```

Comment:
@TomasVotruba I tried using `UpgradeRectorConfigRector`, it seems `paths()` and `skip()` is not changed by `UpgradeRectorConfigRector` so it changed manually.